schemas/territories.json

```diff
@@ -50,9 +50,10 @@
           }
         ],
         "datetime": {
-          "time": "3PM UTC",
+          "time": "3pm UTC",
```

Comment:
Can we add compliancy to iso8601 here?

Reply:
/^(2[0-3]|[01]?[0-9]):?([0-5]?[0-9]):?([0-5]?[0-9])(Z|[+-](?:2[0-3]|[01]?[0-9])(?::?(?:[0-5]?[0-9]))?)$/i
Would be an option for time only.